Source/WebCore/Modules/url-pattern/URLPatternTokenizer.cpp

```diff
@@ -91,10 +91,10 @@ void Tokenizer::addToken(TokenType currentType)
 }
 
 // https://urlpattern.spec.whatwg.org/#process-a-tokenizing-error
-ExceptionOr<void> Tokenizer::processTokenizingError(size_t nextPosition, size_t valuePosition)
+ExceptionOr<void> Tokenizer::processTokenizingError(size_t nextPosition, size_t valuePosition, const String&& callerErrorInfo)
 {
     if (m_policy == TokenizePolicy::Strict)
-        return Exception { ExceptionCode::TypeError, "Invalid token provided to URLPattern tokenizer with strict tokenize policy."_s };
+        return Exception { ExceptionCode::TypeError, makeString("Invalid token provided to URLPattern tokenizer with strict tokenize policy -- "_s, callerErrorInfo) };
```

Comment:
Also we may directly print info here since we will only throw error if the policy is strict (so no need to emphasize error is caused by strict policy):
return return Exception { ExceptionCode::TypeError, WTFMove(errorMessage) };